src/components/profile/index.js

```diff
@@ -2,14 +2,39 @@
 
 /** @jsx jsx */
 import { jsx } from '@emotion/core'
-import React from 'react'
+import * as React from 'react'
+import type { BrowserHistory, HashHistory, MemoryHistory } from 'history'
```

Comment:
What's `history`? This looks like auto-import 💣

Reply:
[aaron/feature/profile-scene 2870b79] :ok_hand: [profile-scene] history type => ReactRouter.ContextRouter